### PR TITLE
I4972 - fixing rating locale bug

### LINKED ISSUE #4972: fixing locale bug that shows up on travis-ci
seeing failed test on ci check:

see: https://travis-ci.org/mozilla/addons-frontend/jobs/377046751

### PR DESCRIPTION
**Pull request by rebmullin**
fixes #4972  - this addresses translation bug that came up in a PR :

https://travis-ci.org/mozilla/addons-frontend/jobs/377046751

### CODE REVIEW

**Comment by muffinresearch** [comment]
@rebmullin if you click through to travis on the test failure you'll see the logs for the dennis-lint command. 

```
>>> Working on: /home/travis/build/mozilla/addons-frontend/locale/ka/LC_MESSAGES/amo.po
E201: invalid variables: %(rating)s
1990:#: src/ui/components/Rating/index.js:67
1991:msgid "Update your rating to %(starRating)s out of 5."
1992:msgstr "შეფასების შეცვლა %(rating)s-ით 5-დან."
E201: invalid variables: %(rating)s
1994:#: src/ui/components/Rating/index.js:71
1995:msgid "Rate this add-on %(starRating)s out of 5."
1996:msgstr "%(rating)s შეფასება 5-დან."
```

In this case as this log shows the problem is in the `locale/ka/LC_MESSAGES/amo.po` file. 

Dennis is catching that the translation key (starting msgid) contains a variable called `%(starRating)s` but the translation (line starting with `msgstr`) is using a variable that doesn't match `%(rating)s`.

To fix this requires editing `locale/ka/LC_MESSAGES/amo.po` and changing the two cases of `%(rating)s` -> `%(starRating)s`.

**Comment by rebmullin** [comment]
thanks @muffinresearch for looking at this. I guess i misunderstood... thanks tho : )

File: locale/ka/LC_MESSAGES/amo.po [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#
```
Comment (bobsilverberg) [inline locale/ka/LC_MESSAGES/amo.po]: This seems like an unnecessary whitespace change.

Comment (rebmullin) [reply locale/ka/LC_MESSAGES/amo.po]: sure thanks
